Docs/Partiels-Manual.md

```diff
@@ -870,7 +871,24 @@ In the upper section of the Plugin Settings window, each item in the list corres
 
 <div style="page-break-after: always;"></div>
 
-## 14. Command-line tools
+## 14. Translations
+
+Partiels was developed in English, but possesses a translation system. Some translations are embedded with the application (for now only the French version is available).
+By default, Partiels uses the language corresponding to the one in use by the operating system if it is available (otherwise, it falls back with English).
+
+It is also possible to dynamically add new translations by adding the translation files in the directory:
+- MacOS: `~/Library/Application Support/Ircam/Partiels/Translations`
+- Linux: `~/Ircam/Partiels/Translations`
+- Windows: `C:\Users\"username"\AppData\Roaming\Ircam\Partiels\Translations` (with C: corresponding to your installation drive)
+
+If you want to create your own custom translations, you can get inspiration from the French translations. You can find it in one of the following directories depending on your system:
+- MacOS: `/Applications/Partiels.app/Resources/Translations`
+- Linux: `/opt/Partiels/Translations` or `~/opt/Partiels/Translations`
+- Windows: `C:\Program Files\Partiels\Translations` or `C:\Program Files (x86)\Partiels\Translations` on older devices (with C: corresponding to your installation drive)
```

Comment:
Add code quote `` around C:  (with `C:` corresponding to your installation drive) to make this more obvious.